package.json

```diff
@@ -586,22 +586,17 @@
                 "title": "%jupyter.command.jupyter.restartkernel.title%",
                 "category": "Jupyter"
             },
-            {
-                "command": "jupyter.notebookeditor.removeallcells",
-                "title": "%jupyter.command.jupyter.notebookeditor.removeallcells.title%",
-                "category": "Jupyter"
```

Comment:
Why are you removing commands?

Reply:
That's a duplicate. It's already there in the file elsewhere.